doc-Infrastructure_Migration_Solution_Guide/cfme/doc/modules/proc_Creating_an_ansible_service_catalog_item.adoc

```diff
@@ -0,0 +1,10 @@
+[id="Creating_an_ansible_service_catalog_item"]
+= Creating an Ansible Service Catalog Item
+
+The CloudForms Inventory must contain at least one Ansible repository, one playbook, and one credential.
+
+.Procedure
+
+. In CloudForms, configure the link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6/html/managing_providers/automation_management_providers#ansible-inside[Ansible repository and credentials].
```

Comment:
1. I think you're safe without referencing CloudForms here and can simply state, "Configure..."